### PR TITLE
feat(styles): update buttons padding, gap and border-radius

### PR DESCRIPTION
**Pull request by imagoiq**
Following https://github.com/swisspost/design-system/issues/1571#issuecomment-1641942510 and discussion from today.

Updated:
* Padding
* Internal gap
* Border-radius

Figma: https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components-Post?node-id=9096%3A36684&mode=dev

### CODE REVIEW

**Review by alizedebray — CHANGES_REQUESTED** [review]
Changes work well with basic/icon buttons, only animated buttons look a bit off:
image

From Dimitri's comment, it doesn't have to be pixel perfect, you can probably just update the `$btn-animation-distance-map` for the text and arrow placement to look like what we currently have.
image

**Comment by alizedebray** [comment]
Can you also update the icon size for large buttons, it is 24px currently instead of 22px in Figma?

**Comment by imagoiq** [comment]
I think this one is ready to be merged, if you could have a look, please @gfellerph @alizedebray @oliverschuerch

**Review by gfellerph — CHANGES_REQUESTED** [review]
Can you add a changeset please?

**Review by gfellerph — APPROVED** [review]
Thx

File: .changeset/polite-radios-train.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@swisspost/design-system-styles": patch
+---
+
+Update button styles: padding, gap and border-radius
```
Comment (alizedebray) [inline .changeset/polite-radios-train.md:5]:
```suggestion
Update button styles: padding, gap and border-radius.
```

**Review by alizedebray — APPROVED** [review]

**Pull request merged**